stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -82,6 +90,19 @@ public fun MessageComposer(
     onCancelAction: () -> Unit = { viewModel.dismissMessageActions() },
     onMentionSelected: (User) -> Unit = { viewModel.selectMention(it) },
     onCommandSelected: (Command) -> Unit = { viewModel.selectCommand(it) },
+    onShowInChannelChecked: (Boolean) -> Unit = { viewModel.setShowInChannel(it) },
+    headerContent: @Composable ColumnScope.(MessageComposerState) -> Unit = {
+        DefaultMessageComposerHeaderContent(
+            messageComposerState = it,
+            onCancelAction = onCancelAction
+        )
+    },
+    footerContent: @Composable ColumnScope.(MessageComposerState) -> Unit = {
+        DefaultMessageComposerFooterContent(
+            messageComposerState = it,
+            onShowInChannelChecked = onShowInChannelChecked
+        )
+    },
     mentionPopupContent: @Composable (List<User>) -> Unit = {
```

Comment:
Maybe we could also add the slot API for the send button, given that we're updating the composer? :]